### PR TITLE
Add sublimino to CODEOWNERS

### PR DESCRIPTION
**Pull request by lumjjb**
As part of https://github.com/cncf/tag-security/issues/879

### CODE REVIEW

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
 # approval for a PR to a co-chair. 
 # This includes major edits or new introductions to the repository.
-*       @ultrasaurus @dshaw @pragashj @lumjjb @TheFoxAtWork @JustinCappos @achetal01 @ashutosh-narkar @anvega @PushkarJ
+*       @ultrasaurus @dshaw @pragashj @lumjjb @TheFoxAtWork @JustinCappos @achetal01 @ashutosh-narkar @anvega @PushkarJ @sublimino
 
 # README.md Owners
-/README.md @ultrasaurus @dshaw @pragashj @lumjjb @TheFoxAtWork @JustinCappos @achetal01 @ashutosh-narkar @anvega @PushkarJ
+/README.md @ultrasaurus @dshaw @pragashj @lumjjb @TheFoxAtWork @JustinCappos @achetal01 @ashutosh-narkar @anvega @PushkarJ @sublimino
 
 # Security assessments
```
Comment (PushkarJ) [inline CODEOWNERS:24]: Did we miss adding @sublimino to these two or this is intentional?

Comment (PushkarJ) [reply CODEOWNERS:24]: By these two I mean assessments and supply-chain-security

**Comment by PushkarJ** [comment]
@lumjjb looks like recent merge has added conflicts to the file for this PR

**Pull request closed without merging**